libs/blocks/caas-config/caas-config.js

```diff
@@ -408,6 +412,9 @@ const TagsPanel = ({ tagsData }) => {
     >
       <${TagSelect} id="orTags" options=${allTags} label="Tags"
     /><//>
+    <label>Advanced Tag Configurations</label>
+    <${Input} label="Use a secondary source for some content types" prop="showSecondarySource" type="checkbox" />
```

Comment:
Is it intended that the input is always visible, while the panel is only visible when context.state.showSecondarySource is set?

Reply:
The panel should only be visible when showSecondarySource is set - I wanted it to be kind of annoying to use this configuration to avoid an author setting it without knowing what it does

Reply:
ok I decided to be less of a baby and just change everything except the parameter sent to chimera-io